buildSrc/build.gradle

```
@@ -0,0 +1,44 @@
+plugins {
+	id 'java'
+	id 'java-gradle-plugin'
+}
+
+repositories {
+	mavenCentral()
+	maven {
+		name "Quilt Repositiory"
+		url "https://maven.quiltmc.org/repository/release"
+	}
+	maven {
+		name "Fabric Repository"
+		url 'https://maven.fabricmc.net'
+	}
+	maven {
+		name "Mojang"
+		url 'https://libraries.minecraft.net/'
+	}
+}
+
+dependencies {
+	implementation("commons-io:commons-io:2.11.0")
+	implementation("de.undercouch:gradle-download-task:4.1.1")
+	implementation("net.fabricmc:stitch:${project.stitch_version}")
+	implementation('org.quiltmc:tiny-remapper:0.7.1')
+	implementation('org.quiltmc:launchermeta-parser:1.0.0')
+	implementation("com.google.guava:guava:30.1.1-jre")
+	implementation("cuchaz:enigma-cli:${project.enigma_version}")
+	implementation("cuchaz:enigma:${project.enigma_version}")
+	implementation("net.fabricmc.unpick:unpick:${project.unpick_version}")
+	implementation("net.fabricmc.unpick:unpick-format-utils:${project.unpick_version}")
+	implementation("net.fabricmc.unpick:unpick-cli:${project.unpick_version}")
+	implementation("net.fabricmc.filament:net.fabricmc.filament.gradle.plugin:0.3.3")
```

```suggestion
	implementation("org.quiltmc:fabric-filament:0.3.3")
```
Should work, I'm not 100% sure